spirv_reflect.c

```diff
@@ -2759,7 +2769,11 @@ static SpvReflectResult ParseInterfaceVariable(
   p_var->decoration_flags = ApplyDecorations(p_type_node_decorations);
   if (p_var_node_decorations != NULL) {
     p_var->decoration_flags |= ApplyDecorations(p_var_node_decorations);
+  } else {
```

Comment:
nit: Would be preferable not to use the java style `if/else`.

Reply:
sorry, what do you mean by "java style"? I just ran this against `clang-format` and this is what it gave

Reply:
Looks like the clang-format isn't as thorough. I'll update that.